site/gatsby-site/src/components/reports/ReportCard.js

```diff
@@ -24,169 +24,214 @@ const ReportCard = ({ item, className = '', incidentId }) => {
-            data-cy={`${expanded ? 'collapse' : 'expand'}-report-button`}
-          >
-            <Trans>{expanded ? 'Collapse' : 'Expand'}</Trans>
-            <svg
-              aria-hidden="true"
-              className="ml-2 -mr-1 w-4 h-4"
-              fill="currentColor"
-              viewBox="0 0 20 20"
-              xmlns="http://www.w3.org/2000/svg"
+              trigger="click"
+              placement="right"
             >
-              {expanded ? (
-                <path
-                  fillRule="evenodd"
-                  d="M3.293 9.707a1 1 0 010-1.414l6-6a1 1 0 011.414 0l6 6a1 1 0 01-1.414 1.414L11 5.414V17a1 1 0 11-2 0V5.414L4.707 9.707a1 1 0 01-1.414 0z"
-                  clipRule="evenodd"
-                ></path>
-              ) : (
-                <path
-                  fillRule="evenodd"
-                  d="M16.707 10.293a1 1 0 010 1.414l-6 6a1 1 0 01-1.414 0l-6-6a1 1 0 111.414-1.414L9 14.586V3a1 1 0 012 0v11.586l4.293-4.293a1 1 0 011.414 0z"
-                  clipRule="evenodd"
-                ></path>
-              )}
-            </svg>
-          </button>
-        </div>
-        {expanded && (
-          <div className="flex w-full flex-row justify-around items-center text-dark-gray">
-            <Actions item={item} />
+              <FontAwesomeIcon
+                icon={faQuestionCircle}
+                style={{ color: 'rgb(210, 210, 210)', cursor: 'pointer' }}
+                className="far fa-question-circle"
+              />
+            </Tooltip>
+          </div>
+          <div data-cy="variant-text_inputs" className="border-1 rounded-lg px-3">
+            <Markdown>{item.text_inputs}</Markdown>
+          </div>
+          <div className="font-bold flex items-center gap-2">
+            <Trans ns="variants">Output and outcomes</Trans>
+            <Tooltip
+              content={
+                <Trans ns="variants">
+                  Provide the relevant details surrounding the incident. Examples include output
+                  text from a chatbot or the nature of injuries sustained from a robot.
+                </Trans>
+              }
+              trigger="click"
+              placement="right"
+            >
+              <FontAwesomeIcon
+                icon={faQuestionCircle}
+                style={{ color: 'rgb(210, 210, 210)', cursor: 'pointer' }}
+                className="far fa-question-circle"
+              />
+            </Tooltip>
+          </div>
+          <div data-cy="variant-text_outputs" className="border-1 rounded-lg px-3">
+            <Markdown>{item.text_outputs}</Markdown>
           </div>
+        </div>
+      )}
+      <div className="flex justify-end mt-4">
+        <button
```

Comment:
It bother me that expanding the text makes the button to collapse it again move off-screen. What if we also made it so that clicking anywhere in the margins of the card would expand/collapse it?

Reply:
Done!